x/stakeibc/keeper/unbonding_records.go

```diff
@@ -198,14 +198,18 @@ func (k Keeper) InitiateAllHostZoneUnbondings(ctx sdk.Context, dayNumber uint64)
 
 func (k Keeper) CleanupEpochUnbondingRecords(ctx sdk.Context) bool {
 	// this function goes through each EpochUnbondingRecord
-	// if any of them don't have any hostZones, then it deletes the record
+	// if any of them don't have any unprocessed hostZoneUnbondings, then it deletes the record
```

Comment:
nit: "if all hostZoneUnbondings on an EpochUnbondingRecord's are processed, the EpochUnbondingRecord is deleted"

Reply:
good call!